src/single-spa-react.js

```diff
@@ -326,17 +341,14 @@ function getElementToRender(opts, props, mountFinished) {
 
   SingleSpaRoot.prototype = Object.create(opts.React.Component.prototype);
   SingleSpaRoot.prototype.componentDidMount = function () {
-    this.mounted = true;
     setTimeout(this.props.mountFinished);
   };
   SingleSpaRoot.prototype.componentWillUnmount = function () {
     setTimeout(this.props.unmountFinished);
   };
   SingleSpaRoot.prototype.render = function () {
     // componentDidUpdate doesn't seem to be called during root.render() for updates
-    if (this.mounted) {
```

Comment:
I was having issues with the `this.mounted` - since it's not on `this.state` maybe new versions of react were not keeping that property there. So I just removed it, since calling `updateFinished` is a no-op if the component has never updated.